packages/rspack/src/compilation.ts

```diff
@@ -611,9 +611,10 @@ export class Compilation {
 	);
 
 	get modules() {
-		return this.getModules().map(item => normalizeJsModule(item));
+		return this.__internal__getModules().map(item => normalizeJsModule(item));
```

Comment:
why not use private field

Reply:
> why not use private field

Cause we don't have `pub(crate)` for TypeScript